src/ui/class-block-editor.php

```diff
@@ -91,16 +83,20 @@ public function enqueue_block_editor_scripts() {
 				'wp-i18n',
 			],
 			DUPLICATE_POST_CURRENT_VERSION,
-			true
+			false
```

Comment:
Double checking: Why and should this be changed to false?

Reply:
`false` makes the script load in the footer, which is good practice since it's not needed by the browser to start rendering the page